contributors/devel/kubemark-guide.md

```diff
@@ -202,10 +234,21 @@ Because there’s no easy way of mocking other managers (e.g. VolumeManager), th
 are not supported in Kubemark (e.g. we can’t schedule Pods with volumes in them
 yet).
 
-As the time passes more fakes will probably be plugged into HollowNodes, but
-it’s crucial to make it as simple as possible to allow running a big number of
-Hollows on a single core.
-
+We currently plan to extend kubemark along the following directions:
+- Progressively introduce more fake components into HollowNodes, keeping it simple
```

Comment:
I don't understand this. We added NPD recently, and I think we don't have any plans to add more in the near future.
Basically, we would like this to be as similar to real clusters as possible, and currently I think we don't have any other components running on each node.